man/bundle-init.ronn

```diff
@@ -16,3 +16,15 @@ created [`Gemfile(5)`][Gemfile(5)].
 
 * `--gemspec`:
   Use the specified .gemspec to create the [`Gemfile(5)`][Gemfile(5)]
+
+## FILES
+
+Included in the default [`Gemfile(5)`][Gemfile(5)]
+generated is the line `# frozen_string_literal: true`. This is a magic comment
+supported for the first time in Ruby 2.3. The presence of this line
+results in all string literals in the file being implicitly frozen.
+Frozen string literals will be the default in Ruby 3.
```

Comment:
AFAIK, Frozen string literal will be default in Ruby 3.0 or not yet determinable. Here `may be` seems more appropriate than `will be`.